puffin/src/lib.rs

```diff
@@ -629,17 +629,47 @@ macro_rules! current_function_name {
 
 #[doc(hidden)]
 #[inline]
```

Comment:
this should not be inlined anymore as it will bloat the callsites and instruction cache, actually let's make it never inline as we cache it.

```suggestion
#[inline(never)]
```

Reply:
good catch!